task03-one-piece-website/index.html

```diff
@@ -329,5 +329,13 @@ <h3 class="mistObject__card__name">Gomu Gomu No Mi</h3>
         </a>
       </div>
     </div>
+
+    <div class="footer">
+      <img src="./images/calavera.svg" />
```

Comment:
in case you need to add different images depends the devices I recommend you use picture https://www.w3schools.com/tags/tag_picture.asp